src/main/java/edu/harvard/iq/dataverse/dataaccess/FileAccessIO.java

```diff
@@ -415,8 +415,8 @@ public void deleteAllAuxObjects() throws IOException {
         }
         
     }
-    
-    
+
```

Comment:
In general, mixing a lot of reformatting with big changes can be confusing, but I would definitely not touch files in the final PR where you don't make any changes. (Apologies if you were planning to clean this up later already - just making any comments that might be useful as I go through looking for more substantive things.)